Admonition Theme Import - Module not found

### Have you read the Contributing Guidelines on issues?

- [X] I have read the [Contributing Guidelines on issues](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#reporting-new-issues).

### Description

On the [Admonitions page](https://docusaurus.io/docs/markdown-features/admonitions#usage-in-jsx) it states the component can be imported into a jsx file using:`import Admonition from '@theme/Admonitions';`.

However this is incorrect and causes a error:`Module not found: Error: Can't resolve '@theme/Admonitions' in '/sandbox/src/components'`

The import should be `import Admonition from '@theme/Admonition';` **without the 's'** and this works without error.

### Self-service

- [ ] I'd be willing to address this documentation request myself.

--- a/website/docs/guides/markdown-features/markdown-features-admonitions.mdx
+++ b/website/docs/guides/markdown-features/markdown-features-admonitions.mdx
@@ -83,8 +83,8 @@ Hello world
 :::
 
 <!-- Prettier changes this -->
-::: note 
-Hello world 
+::: note
+Hello world
 :::
 
 <!-- to this -->
@@ -146,10 +146,10 @@ import TabItem from '@theme/TabItem';
 
 ## Usage in JSX
 
-Outside of Markdown, you can use the `@theme/Admonitions` component to get the same output.
+Outside of Markdown, you can use the `@theme/Admonition` component to get the same output.
 
 ```jsx title="MyReactPage.jsx"
-import Admonition from '@theme/Admonitions';
+import Admonition from '@theme/Admonition';
 
 export default function MyReactPage() {
   return (

--- a/website/versioned_docs/version-2.0.0-beta.14/guides/markdown-features/markdown-features-admonitions.mdx
+++ b/website/versioned_docs/version-2.0.0-beta.14/guides/markdown-features/markdown-features-admonitions.mdx
@@ -83,8 +83,8 @@ Hello world
 :::
 
 <!-- Prettier changes this -->
-::: note 
-Hello world 
+::: note
+Hello world
 :::
 
 <!-- to this -->
@@ -146,10 +146,10 @@ import TabItem from '@theme/TabItem';
 
 ## Usage in JSX
 
-Outside of Markdown, you can use the `@theme/Admonitions` component to get the same output.
+Outside of Markdown, you can use the `@theme/Admonition` component to get the same output.
 
 ```jsx title="MyReactPage.jsx"
-import Admonition from '@theme/Admonitions';
+import Admonition from '@theme/Admonition';
 
 export default function MyReactPage() {
   return (

